saleor/payment/utils.py

```diff
@@ -433,3 +433,29 @@ def price_to_minor_unit(value: Decimal, currency: str):
     number_places = Decimal("10.0") ** precision
     value_without_comma = value * number_places
     return str(value_without_comma.quantize(Decimal("1")))
+
+
+def get_channel_slug_from_payment(payment: Payment) -> Union[str, None]:
+    if payment.checkout:
+        channel_slug = payment.checkout.channel.slug
+    elif payment.order:
+        channel_slug = payment.order.channel.slug
+    else:
+        channel_slug = None  # type: ignore
```

Comment:
Why `# type: ignore` is needed here?

Reply:
`saleor/payment/utils.py:444: error: Incompatible types in assignment (expression has type "None", variable has type "str")` because of mypy :( anyway, refactored  and type ignore not needed anymore